tests/suites/0_stateless/20+_others/20_0001_show.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. "$CURDIR"/../../../shell_env.sh
+
+
+
+stmt "drop table if exists default.tc;"
+stmt 'create table default.tc(id int, Id1_ int, "id2)" int, "ID4" int);'
```

Comment:
consider adding a name with ", for example "ID""5"

Reply:
Done